### DISCUSSION TITLE
Access API without ollama (external Groq)

### DISCUSSION THREAD

**Discussion by dewrama** [open-webui/open-webui#4901] [Q&A]
I am able to hit the API endpoint with ollama installation (localhost:3030/ollama/api/generate). What if I don't use ollama? Is there another API I can use with the the generated API key (sk-111111111)? I would like a way to access open-webui endpoint (not olllama dependent) with authentication. Thanks.

**Comment by dewrama** [comment]
I figured this out. You can use openai/chat/completions (also ollama/v1/chat/completions). Tested it both with curl and fastapi.

**Reply by DmacMcgreg** [reply]
@dewrama could you give the full cURL request? I cannot get it to work.
